src/hooks/usePrevious.js

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useRef } from 'react'
+
+function usePrevious (value) {
+  const ref = useRef()
```

Comment:
```suggestion
  const ref = useRef(null)
```
maybe is better initialize `userRef` with null